app/src/main/java/org/gem/indo/dooit/helpers/InvalidTokenHandler/OpenLoginHandler.java

```diff
@@ -0,0 +1,31 @@
+package org.gem.indo.dooit.helpers.InvalidTokenHandler;
```

Comment:
Try and keep to the naming conventions we have in place. Keep package names short and lower case.